### PR TITLE
Fix warning unused parameter samples

### PR DESCRIPTION
**Pull request by Bogumil-Sapinski-Mobica**

### CODE REVIEW

File: samples/core/enumopencl/main.cpp [hunk 1]
```diff
@@ -67,6 +67,9 @@ static cl_int PrintDeviceInfoSummary(const std::vector<cl::Device> devices)
 
 int main(int argc, char** argv)
 {
+    (void)argc;
+    (void**)argv;
```
Comment (alycm) [inline samples/core/enumopencl/main.cpp:71]:
```suggestion
    (void)argv;
```

While your suggested change is not wrong just alway cast to `(void)` is the idiomatic way to make this as unused.

File: samples/core/enumopencl/main.cpp [hunk 2]
```diff
@@ -67,6 +67,9 @@ static cl_int PrintDeviceInfoSummary(const std::vector<cl::Device> devices)
 
 int main(int argc, char** argv)
```
Comment (alycm) [inline samples/core/enumopencl/main.cpp:68]:
```suggestion
int main(int, char**)
```

An alternative way to eliminate the unused warning, drop the unused names.

Do not take both of my suggestions, just one, either is okay with me.

Comment (Bogumil-Sapinski-Mobica) [reply samples/core/enumopencl/main.cpp]: Done as suggested.

**Comment by Bogumil-Sapinski-Mobica** [comment]
CI fails on SFML which was not changed in this PR. I kindly ask you to view and solve this issue as I do not have an access to CI process internals in order to fix it.

**Review by bashbaug — APPROVED** [review]
LGTM!

**Comment by bashbaug** [comment]
Merging as discussed in the September 12th teleconference.  The CI failures are unrelated to these changes.

**Pull request merged**